openrouteservice/src/main/java/org/heigit/ors/routing/graphhopper/extensions/storages/SpeedStorage.java

```diff
@@ -0,0 +1,135 @@
+package org.heigit.ors.routing.graphhopper.extensions.storages;
+
+import com.graphhopper.routing.util.FlagEncoder;
+import com.graphhopper.storage.DataAccess;
+import com.graphhopper.storage.Directory;
+import com.graphhopper.storage.Graph;
+import com.graphhopper.storage.GraphExtension;
+
+/**
+ * Simple storage designed to hold edgeID - direction - speed
+ * Speeds should be in kph
+ * Indexed by edgeIds
+ *
+ * @author Hendrik Leuschner
+ */
+public class SpeedStorage implements GraphExtension {
+    private static final long BYTE_COUNT = 2; //One byte for forward speed, one byte for backward speed, 2 x 4 bytes for time stamp in unix time minutes
```

Comment:
Comment does not reflect code.

Reply:
Resolved. Wrong comment from other storage.